### ISSUE TITLE
Use dict rather than RequestContext in render method

### ISSUE THREAD

**Issue by phlax** [translate/pootle#5442]
something in pootle_store.views is triggering this warning

```
RemovedInDjango110Warning: render() must be called with a dict, not a RequestContext.
  RequestContext(self.request, context))
```

**Comment by dwaynebailey** [comment]
Fixed in 1a1fb2ef27d34cf262bd617f73c57f1a7a0258f1

**Comment by dwaynebailey** [comment]
There are some other cases fixed in the Django 1.10 cleanup, so lets keep this open for now

**Comment by dwaynebailey** [comment]
It seems this is the one I was waiting for 87612a425a35b320a11470c76a58f7f88b519255 in `render_to_string`
